src/app/tvshows/tvshows-tvdata.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input } from '@angular/core';
+import { TvShowData } from './store/tvshows.models';
+
+@Component( {
+  selector: 'app-tvshow-poster',
+  template: `
+  <table border="1" style="display:inline-block">
```

Comment:
How come your using a table for this? I don't see why it's necessary to use a able. This is also an anti-pattern to use table for layout only.

Reply:
This is temporary, I do plan to replace this with angular-material cards.    Using tables is lazy of me though...  I can fix this quickly.